Content.Shared/Blocking/BlockingUserSystem.cs

```diff
@@ -54,7 +54,7 @@ private void OnAnchorChanged(EntityUid uid, BlockingUserComponent component, ref
 
     private void OnDamageChanged(EntityUid uid, BlockingUserComponent component, DamageChangedEvent args)
     {
-        if (component.BlockingItem != null)
+        if (component.BlockingItem != null && args.DamageIncreased)
```

Comment:
shouldnt this be getting checked on the subscription for the blocking item? that way if you somehow directly heal the shield, it won't also break